service/templates/projects.html

```diff
@@ -126,6 +126,14 @@ <h2> Settings for [[displayName]]</h2>
         </div>
       </div>
     </div>
+    <div id="version-control-enabled" class="form-group">
+      <div class="col-lg-6">
+        <input type="checkbox" id="version-control-enabled-checkbox" ng-model="settingsFormData.version_control_enabled" />
+        <label for="version-control-enabled-checkbox">Enable Config Version Control</label>
+<!--        todo: write documentation and provide link below-->
+        <div class="muted small">Allow defining of <a href="">these properties</a> on this page in config yaml rather than exclusively in this UI.</div>
```

Comment:
Can we move this lower on the project page? Right now this is right below Enabled/Disabled which is the most important toggle on the page, whereas this feature is only applicable to some teams. I think a new section similar to Test Results may be reasonable.

Reply:
Sure thing. I can't reply to your second comment for some reason, maybe we should keep this PR open til we have some docs available? I am working on them right now

Reply:
Whichever you prefer is fine 👍